appeals/web/src/server/lib/mappers/appeal.mapper.js

```diff
@@ -1066,6 +1066,39 @@ export async function initialiseAndMapAppealData(
 		}
 	};
 
+	const appealHasWithdrawalDocuments =
+		appealDetails?.withdrawal?.withdrawalFolder?.documents?.filter(
+			(document) => document.latestDocumentVersion?.isDeleted === false
+		).length;
+
+	const appealWithdrawalActionItem =
+		appealDetails.appealStatus === 'withdrawn' && appealHasWithdrawalDocuments
```

Comment:
This should reference `import { APPEAL_CASE_STATUS } from 'pins-data-model';`

Reply:
done